kaggle_workshop/Student Exam Performance Workshop.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>Good hint! </p>

 _Reply via <a href='https://app.reviewnb.com/georgetown-analytics/XBUS-501-01.Software-Engineering-for-Data/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>